### PR TITLE
Update the text to be a decent starting point

### PR DESCRIPTION
**Pull request by sondreb**

### CODE REVIEW

File: src/Blockcore.Website/ClientApp/src/app/app.component.html [hunk 1]
```diff
@@ -6,50 +6,49 @@
...
+               Blockcore is a platform that enables you to build your own blockchains.
+            </h2>
+            <p>
+               Blockcore is the foundation for realizing blockchains and includes core functionality to create your own custom blockchain with a lot of tooling supporting your blockchain.
+            </p>
+            <p>
+               The tooling as part of the Blockcore platform includes; Block Explorer, Block Indexing, Block Analytics, Wallets, API Wallet Service, Developer Tooling, Documentation and more.
+            </p>
+            <p>
+               Blockcore is currently under development and not yet available for production use. If you are interested in our platform, please engadge with our community on <a href="https://discord.gg/xa5BxWr">Discord</a>.
```
Comment (dangershony) [inline src/Blockcore.Website/ClientApp/src/app/app.component.html]: Why not ready for production?

What do you mean production in what sense? there are already a few cryptocurrencies built on top of the code base

Comment (sondreb) [reply src/Blockcore.Website/ClientApp/src/app/app.component.html]: With that I mean one would have to fork the FN to make a chain today, but I think we should prepare NuGet and a reference implementation that does not base on forked source code. The Indexer was just refactored, not fully ready yet. The Explorer is being refactored now, etc. It was just to be a warning until we're more ready to have random devs build new chains on the platform.

Comment (dangershony) [reply src/Blockcore.Website/ClientApp/src/app/app.component.html]: This I agree, it's just that the term production ready doesn't feel correct.
Also blockcore has many products in it some ,more mature then others, its probably more correct to describe readyness on each repo itself.
How about:
>  Blockcore is under continues development and constantly being improved and follow the latest industry tech.

File: src/Blockcore.Website/ClientApp/src/app/app.component.html [hunk 2]
```diff
@@ -6,50 +6,49 @@
...
+               Blockcore is a platform that enables you to build your own blockchains.
+            </h2>
+            <p>
+               Blockcore is the foundation for realizing blockchains and includes core functionality to create your own custom blockchain with a lot of tooling supporting your blockchain.
+            </p>
+            <p>
+               The tooling as part of the Blockcore platform includes; Block Explorer, Block Indexing, Block Analytics, Wallets, API Wallet Service, Developer Tooling, Documentation and more.
+            </p>
+            <p>
+               Blockcore is under continuous development and constantly being improved and follow the latest industry tech. If you are interested in our platform, please engadge with our community on <a href="https://discord.gg/xa5BxWr">Discord</a>.
```
Comment (dangershony) [inline src/Blockcore.Website/ClientApp/src/app/app.component.html]:
```suggestion
               Blockcore is under continuous development to stay at the cutting edge of Bitcoin and blockchain technology.   
               We invite you to engage with our community on our <a href="https://discord.gg/xa5BxWr">Discord</a>.
```

**Pull request merged**